pkgs/by-name/ma/mailsy/package.nix

```
@@ -0,0 +1,32 @@
+{
+  buildNpmPackage,
+  fetchFromGitHub,
+  lib,
```

This is more of a cosmetic thing (as far as I know), but `lib` is usually the first input of the derviation.

```suggestion
  lib,
  buildNpmPackage,
  fetchFromGitHub,
```